src/cmd-test-coreos-installer

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# Automate an end-to-end run of coreos-installer with the metal image, which then
+# boots and writes a success message to a virtio-serial port, which we read on the host.
+set -euo pipefail
+
+set -x
+
+dn=$(dirname "$0")
+# shellcheck source=src/cmdlib.sh
+. "${dn}"/cmdlib.sh
+
+build=${1:-latest}
+
+arch=$(arch)
+
+builddir=builds/${build}/${arch}
+buildmeta=${builddir}/meta.json
+
+buildid=$(jq -er .buildid < ${buildmeta})
+metalimg=$(jq -er .images.metal.path < ${buildmeta})
+instkernel=$(jq -er .images.kernel.path < ${buildmeta})
+instinitramfs=$(jq -er .images.initramfs.path < ${buildmeta})
```

Comment:
In light of https://github.com/coreos/fedora-coreos-tracker/issues/203, we'll want to at least for now make this configurable to use the live versions if possible. The kargs interface should be the same otherwise so we shouldn't have to change anything else.

We can do this in a follow-up though if you prefer!

Reply:
Yeah I just want to get *any* coverage of the installer path for RHCOS right now.